api/Animation.json

```diff
@@ -1599,58 +1161,20 @@
             "edge": {
               "version_added": false
             },
-            "firefox": [
-              {
-                "version_added": "75",
-                "notes": "Currently only the getter is supported"
-              },
-              {
-                "version_added": "63",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.animations-api.timelines.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              {
-                "version_added": "48",
-                "version_removed": "63",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.animations-api.core.enabled",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "This property is supported in Firefox 48 but is read-only. It became writable in Firefox 49."
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": "63",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.animations-api.timelines.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              {
-                "version_added": "48",
-                "version_removed": "63",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.animations-api.core.enabled",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "This property is supported in Firefox 48 but is read-only. It became writable in Firefox 49."
-              }
-            ],
+            "firefox": {
+              "version_added": "75",
+              "notes": "Currently only the getter is supported"
+            },
+            "firefox_android": {
+              "version_added": "63",
+              "flags": [
```

Comment:
Is the existing data really correct here, or was the Firefox entry updated at some point while forgetting the Firefox for Android entry?

Reply:
I suspect it's the latter. https://bugzilla.mozilla.org/show_bug.cgi?id=1619178

Reply:
OK, so let's merge this PR and then I'll send a separate one to fix this.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/9436.